framework/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
+- Enh: migrate command can create foreign keys.
```

Comment:
I didn't found any issue open or closed requesting for this. Should I use this same PR number?

also I am working right now on giving support for `useTablePrefix` option in the generator. I am having issues with the FK names it will use. is `fk-{{%table1}}-{{%table2}}` acceptable?